algolia.js

```diff
@@ -93,11 +88,25 @@ async function index() {
 
   const raw = fs.readFileSync('dist/pages.json', 'utf-8');
   /** @type {AlgoliaItem[]} */
-  const algoliaData = JSON.parse(raw).map((e) => {
-    // Set date of when object is being added to algolia.
-    e.indexedOn = indexedOn.getTime();
-    return e;
-  });
+  const algoliaData = JSON.parse(raw)
+    .map((/** @type {AlgoliaItem} */ e) => {
```

Comment:
"e" is usually used for "event". What does it stand for here? Can you use full name instead?